dist/scripts/rx-data-table.js

```diff
@@ -417,28 +417,32 @@ app.directive('rxDataTable', function ($http, $timeout, $document, $filter, $par
             });
 
             scope.scoreTooltipTemplate = function (row) {
+                var tempWeight = (row.score.elapsedTime - row.score.slMaxResponseTime) / 600;
```

Comment:
@JoseVillarreal  new score formula doesn't seem to return `slMaxResponseTime`. 
Seems`maxResponseTime` = `slMaxResponseTime` in new formula.
Need to check with Samatha if we can `maxResponseTime`